jarvis.py

```diff
@@ -2520,6 +2531,67 @@ def vpn_checker():
                     "integrations will work with VPN enabled.")
     else:
         return '.'.join(ip_address.split('.')[0:3])
+                         
+                         
+def timer(converted):
+    speaker.say("For how many minutes?")
+    speaker.runAndWait()
+    timer0 = listener(3, 5)
+    converted.replace('for', 'minutes', 'minute', 'in', '')
+
+    timer1 = float(timer0)
+    timer2 = timer1 * 60
+    speaker.say(f'I will remind you in {timer2} seconds')
+
+    time.sleep(timer2)
+    speaker.say('Your time has been finished sir')
+                         
+                         
+def facebook(profile):
```

Comment:
I have this as a separate project but `GraphAPI` doesn't work the way you expect, the APIs are not persistent and keeps expiring often, so I will not be integrating `facebook` anytime soon